src/Online_Bookstore/Dashboard/SetReviewInLocalStorage.js

```diff
@@ -0,0 +1,51 @@
+
+exports.handleSubmit = ({id,rating,comment,booksData}) => {
+    const getBookData = window.localStorage.getItem("user_login");
+    const userData = JSON.parse(getBookData);
+    const userName = userData[0].name;
+    console.log("HERE",booksData.isbn13);
+    console.log(id);
+    if (window.localStorage.getItem("reviewBooks")) {
+      const reviewData = JSON.parse(window.localStorage.getItem("reviewBooks"));
+      if (reviewData[userName]) {
```

Comment:
This code block doesn't make any sense. Why the property `reviewData[userName][id]` has been set with values in both if-else blocks in the same way? on line no 11 and then on line no 21, when it has been already emptied on line no 20?

Refactor accordingly

Reply:
ma'am I have done this becoz if there is no book reviewed by any user then we have to first intialize the new object